docs/release-notes/release-2023-06-06.md

```diff
@@ -0,0 +1,150 @@
+---
+title: June 2023
+---
+
+# Release Notes June 2023 (2023-06-06)
+
+:::danger
+This release contains **breaking changes**
+:::
+
+## New Features
+
+- New decimal types
+- Support for documentation comments
+- New `valueof` type constraint modifier for type literals
+- Lazy evaluation of emitter configuration
+- Signature Help for Template Arguments and other IDE improvements
+- Json Schema Emitter
+
+### New Decimal Types
+
+Added support for `decimal` and `decimal128` types
+
+```typespec
+model Foo {
+  price: decimal;
+  shortPrice: decimal128;
+}
+```
+
```

Comment:
```suggestion
These types represent values stored in decimal format which are often used in financial calculations to avoid rounding errors introduced by binary floating point formats. `decimal` can store values of any size and precision, while `decimal128` corresponds to the 128 bit decimal floating point format described by IEEE 754.
```